### PR TITLE
Feature/dtd 170 request interest for drier mvp

### PR DESCRIPTION
**Pull request by rogerball**

### CODE REVIEW

**Review by victorarbuesmallada — APPROVED** [review]
LGTM

File: src/test/resources/features/ifs/GetDebtForDrierCase.feature [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  Scenario: DRIER debt Zero Amount Edge Case
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      | 0      | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with
+      | dailyInterest | totalInterest | intRate | totalAmountToPay | totalAmountWithInterest | numberChargeableDays |
+      | 0             | 0             | 1       | 0                | 0                       | 8          |
+
+  Scenario: DRIER debt Amount is zero (Edge Case)
```
Comment (kiranpl) [inline src/test/resources/features/ifs/GetDebtForDrierCase.feature]: This looks like a duplicate of the above scenario

**Review by thadeusssewanyana — APPROVED** [review]

File: src/test/resources/features/ifs/GetDebtForDrierCase.feature [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    Then the ifs service will respond with '/amount' missing or invalid
+
+  Scenario: DRIER debt empty Amount (Edge Case)
+    Given a debt item
+      | amount | dateAmount | dateCalculationTo | regime | chargeType | interestBearing |
+      |        | 2021-03-01 | 2021-03-08        | DRIER  | NI         | true            |
+    When the debt item is sent to the ifs service
+    Then the ifs service will respond with '/amount' missing or invalid
+
+  Scenario: DRIER debt invalid entry in Date Amount (Edge Case)
```
Comment (kiranpl) [inline src/test/resources/features/ifs/GetDebtForDrierCase.feature:81]: In the scenario description, "Date Amount" doesn't make sense. Please change to "Date value"

**Review by roccobruno — COMMENTED** [review]
Looks ok, but in none of those scenarios there is a change in a config rule. I.e. the scenario of not bearing interests for DRIER can pass if we add a step that changes to rule for DRIER.

**Review by thadeusssewanyana — APPROVED** [review]

**Review by kiranpl — APPROVED** [review]
